pimpmycause/pimpmycause/templates/profiles/login.html

```diff
@@ -19,7 +19,7 @@ <h1 class="pnl__title">Log In</h1>
                             </div>
                             <div class="row">
                                 <input class="button round right" type="submit" value="Sign in">
-                                <a href="create-account.html" class="underline txt-right">Not Registered? Start here</a>  
+                                <a href="register.html" class="underline txt-right">Not Registered? Start here</a>  
```

Comment:
Since we moved to Django the urls in django are different, the path for a login is same as from the homepage; https://github.com/womenhackfornonprofits/pimp-my-cause-django/blob/master/pimpmycause/pimpmycause/templates/index.html#L22